providers/social.go

```diff
@@ -113,16 +113,23 @@ func (s *Social) Init(handler tap.IdentityHandler, profile tap.Profile, config [
 			gothProviders = append(gothProviders, bitbucket.New(provider.Key, provider.Secret, s.getCallBackURL(provider.Name)))
 
 		case "openid-connect":
+            discoverURL, err := url.Parse(provider.DiscoverURL)
+            if err != nil {
+                return err
+            }
+
 			gProv, err := openidConnect.New(provider.Key, provider.Secret, s.getCallBackURL(provider.Name), provider.DiscoverURL)
 			if err != nil {
 				return err
 			}
 
-			if provider.BustedProvider != "" {
-				oauth2.RegisterBrokenAuthHeaderProvider(provider.BustedProvider)
+            // See https://godoc.org/golang.org/x/oauth2#RegisterBrokenAuthHeaderProvider
+			if provider.DisableAuthHeader {
+                cleanDomain := strings.Replace(provider.DiscoverURL, discoverURL.RequestURI(), "", -1)
```

Comment:
This does not work with Okta, as the discover domain is different from the actual callback domain (callback: `https://dev-xxxx.oktapreview.com` and discover: `https://dev-xxxx-admin.oktapreview.com/...`), we also should make this exception as a string for other providers as this may not be isolated to Okta.

Reply:
(Suggest reverting to string and changing the name to something like "DisableAuthHeaderProviderDomain").